### DISCUSSION TITLE
Can we add an autosave feature of annotations to an existing PDF?

### DISCUSSION THREAD

**Discussion by wesmosis** [sumatrapdfreader/sumatrapdf#3208] [Ideas]
e.g. every 5 or 10 or 15 minutes

**Comment by GitHubRulesOK** [comment]
Not a good idea as each save bloats the file by a considerable amount, every add and delete (hidden not redacted) saved, needs a reindex added thus overheads quickly mount, that's not SumatraPDF or MuPDF doing, it is the way Adobe initially designed appendages ( its not an unreasonable method, but very heavy on file storage) Its proportional to number of edits, so for a small file the highlights could be larger than the source. Since you know your rate of edits one per hour or 1000 per hour its best you save as often as experience dictates.

**Reply by wesmosis** [reply]
Gotcha, thanks for the prompt response!
